packages/ui-core/src/components/color-scale/color-scale.styles.ts

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const ColorCard = styled.div<{ background: string }>`
+  width: 100%;
+  height: 6px;
+  background: ${props => props.background};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 13px;
+`;
+
+export const ColorGradient = styled.div<{
```

Comment:
Instead of passing collection of colors and destructing it - please pass `startColor` and `endColor` as CSS component shouldn't be responsible for data extraction.